seedTestFiles/seedInsertTestUnique1k.js

```diff
@@ -0,0 +1,92 @@
+const faker = require('faker');
+const mongoose = require('mongoose');
+const Stock = require('../databases/database-mongoose/SchemaMongoose');
+
+mongoose.connect(process.env.MONGODB_URI || 'mongodb://localhost/stocks');
```

Comment:
This is great, make sure .env file doesn't get uploaded if that's where you're choosing to store it!